core/modules/commands/rendertiddlers.js

```diff
@@ -35,22 +36,33 @@ Command.prototype.execute = function() {
 		wiki = this.commander.wiki,
 		filter = this.params[0],
 		template = this.params[1],
-		pathname = path.resolve(this.commander.outputPath,this.params[2]),
+		outputPath = this.commander.outputPath,
+		pathParam = this.params[2],
+		rootPath = path.resolve(outputPath,pathParam),
 		type = this.params[3] || "text/html",
 		extension = this.params[4] || ".html",
 		deleteDirectory = (this.params[5] || "") != "noclean",
 		tiddlers = wiki.filterTiddlers(filter);
+	var parser = wiki.parseTiddler(template),widgetNode = wiki.makeWidget(parser);
```

Comment:
Unless I add these lines the condition directly below ( if(link.getExportFolder){ ) is always false. How can i enable the plugin without calling those two functions?